packages/curve-ui-kit/src/shared/ui/tanstack-table.overrides.d.ts

```diff
@@ -0,0 +1,7 @@
+import '@tanstack/react-table'
+
+declare module '@tanstack/table-core' {
```

Comment:
Is this still needed or were you testing something? I find it odd you'd need to manually interfere with tanstack react typings.

Reply:
Yeah this came from their docs actually. It's the way to customize the metadata type